app/mailers/user_notifier_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class UserNotifierMailer < ApplicationMailer
```

Comment:
Could we change the name to describe what's being sent more? The email being sent isn't technically a notification